k8s/values.yaml

```diff
@@ -36,21 +36,21 @@ applications:
       hosts:
         - host: penncourseplan.com
           paths: ["/"]
-  - name: alert
-    image: pennlabs/penn-courses-backend
-    secret: penn-courses
-    ingress:
-      hosts:
-        - host: penncoursealert.com
-          paths: ["/"]
-    extraEnv:
-      - name: ROOT_URLCONF
-        value: PennCourses.urls.pca
-      - name: DJANGO_SETTINGS_MODULE
-        value: PennCourses.settings.production
-  - name: alert-beta
+  # - name: alert
+  #   image: pennlabs/penn-courses-backend
+  #   secret: penn-courses
+  #   ingress:
+  #     hosts:
+  #       - host: penncoursealert.com
+  #         paths: ["/"]
+  #   extraEnv:
+  #     - name: ROOT_URLCONF
+  #       value: PennCourses.urls.pca
+  #     - name: DJANGO_SETTINGS_MODULE
+  #       value: PennCourses.settings.production
```

Comment:
We can probably just delete these lines

Reply:
in case we need to switch back in a hurry, I wanted to keep them in.